site/pages/api/delete-disruption.api.ts

```diff
@@ -19,7 +19,7 @@ const deleteDisruption = async (req: NextApiRequest, res: NextApiResponse): Prom
             );
         }
 
-        const disruption = await getDisruptionById(id, session.orgId);
+        const disruption = await getDisruptionById(id, session.orgId, req.query.template === "true");
```

Comment:
Currently if you delete a template it takes you back to the dashboard. Should this take us back to /view-all-templates page instead?

Reply:
done